test/unit/org/apache/cassandra/index/sai/memory/AbstractTrieMemtableIndexTest.java

```diff
@@ -18,286 +18,24 @@
 
 package org.apache.cassandra.index.sai.memory;
 
-import java.nio.ByteBuffer;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeMap;
-import java.util.stream.Collectors;
-
-import org.junit.Before;
+import org.apache.cassandra.cql3.CQLTester;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
-import org.junit.Test;
 
-import org.apache.cassandra.cql3.CQLTester;
-import org.apache.cassandra.cql3.Operator;
-import org.apache.cassandra.db.Clustering;
-import org.apache.cassandra.db.ColumnFamilyStore;
-import org.apache.cassandra.db.DecoratedKey;
-import org.apache.cassandra.db.PartitionPosition;
-import org.apache.cassandra.db.marshal.Int32Type;
-import org.apache.cassandra.db.memtable.AbstractAllocatorMemtable;
-import org.apache.cassandra.db.memtable.AbstractShardedMemtable;
-import org.apache.cassandra.dht.AbstractBounds;
-import org.apache.cassandra.dht.BootStrapper;
-import org.apache.cassandra.dht.Bounds;
-import org.apache.cassandra.dht.ExcludingBounds;
-import org.apache.cassandra.dht.IPartitioner;
-import org.apache.cassandra.dht.IncludingExcludingBounds;
-import org.apache.cassandra.dht.Range;
-import org.apache.cassandra.index.sai.IndexContext;
-import org.apache.cassandra.index.sai.QueryContext;
-import org.apache.cassandra.index.sai.SAITester;
-import org.apache.cassandra.index.sai.iterators.KeyRangeIterator;
-import org.apache.cassandra.index.sai.plan.Expression;
-import org.apache.cassandra.index.sai.utils.PrimaryKey;
-import org.apache.cassandra.index.sai.utils.TypeUtil;
-import org.apache.cassandra.locator.TokenMetadata;
-import org.apache.cassandra.schema.MockSchema;
-import org.apache.cassandra.schema.TableMetadata;
-import org.apache.cassandra.service.StorageService;
-import org.apache.cassandra.utils.FBUtilities;
-import org.apache.cassandra.utils.Pair;
-import org.apache.cassandra.utils.bytecomparable.ByteComparable;
-import org.apache.cassandra.utils.bytecomparable.ByteSource;
-import org.apache.cassandra.utils.concurrent.OpOrder;
+import org.apache.cassandra.config.Config;
+import org.apache.cassandra.db.memtable.TrieMemtable;
+import org.apache.cassandra.io.compress.BufferType;
 
 import static org.apache.cassandra.config.CassandraRelevantProperties.MEMTABLE_SHARD_COUNT;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
 
-@Ignore
-public abstract class AbstractTrieMemtableIndexTest extends SAITester
+public class AbstractTrieMemtableIndexTest extends TrieMemtableIndexTestBase
```

Comment:
not abstract anymore

Reply:
Yes, thanks, I missed that 👍🏻 

Also, it looks like just removing this (leftover) line makes the tests pass:
```
        CQLTester.setUpClass();
```

Reply:
Oh, actually there were commits from CNDB-11322 and CNDB-11353 made to `main-5.0` that made this abstract and introduced new subclasses `OffHeapTrieMemtableIndexTest` and `OnHeapTrieMemtableIndexTest`:
https://github.com/datastax/cassandra/commit/3c39fb4770b3c5d9d2e129541c3ef989b55fca74